BulletHell/Models/Bullet.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using BulletHell.Configurations;
+using BulletHell.Interfaces;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+
+namespace BulletHell.Models;
+
+public class Bullet(Vector2 startPosition, Vector2 direction, ISpriteHelper sprite)
```

Comment:
```suggestion
public class Bullet(Vector2 startPosition, Vector2 direction, ISpriteHelper sprite) : IDisposable
```
Funderar på om Bullet ska ha IDisposable och när dem tas bort från [bullet-listan i game1.cs](https://github.com/JojoSlice/BulletHell/pull/21/files#diff-970f69f07f74ec74f849760fbabf0333df2d797398ae01749c71599b7da7d91fR79) så kör man dispose också på dem som är `!IsAlive` för att rensa dem från minnet